lib/ZwaveClient.ts

```diff
@@ -2025,6 +2023,9 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 		this._updateControllerStatus('Driver ready')
 
 		try {
+			// this must be done only after driver is ready
+			await this._scheduledConfigCheck()
```

Comment:
I wouldn't await here, just fire and forget.
```suggestion
			this._scheduledConfigCheck().catch(() => { /* ignore */ });
```

Or at least do it after attaching the event handlers.